packages/page-accounts/src/Accounts/BannerExtension.tsx

```diff
@@ -26,6 +26,7 @@ function BannerExtension (): React.ReactElement | null {
   const { t } = useTranslation();
   const { hasInjectedAccounts } = useApi();
   const upgradableCount = useExtensionCounter();
+  const phishing = useRef<string>(t<string>('Since some extensions, such as the polkadot{.js} extension protects you against all community-reported phishing sites, there are valid reasons to use them, even if you are not storing accounts in it.'));
```

Comment:
```suggestion
  const phishing = useRef<string>(t<string>('Since some extensions, such as the polkadot{.js} extension, protects you against all community reported phishing sites, there are valid reasons to use them for additional protection, even if you are not storing accounts in it.'));
```